qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -0,0 +1,79 @@
+# Notes:
+#    For ppc64/ppc64le, please manually specify cpu_model in your environment
+- cpu_device_hotpluggable: install setup image_copy unattended_install.cdrom
+    only ppc64 ppc64le x86_64
+    Windows:
+        only Win2008.x86_64 Win2012.x86_64 Win2016.x86_64 Win2019.x86_64
+    no ovmf
```

Comment:
Any comment for this line?

Reply:
After discussing with feature owner, OVMF's vCPU hotpluggable is a new feature in the future. So skip test it.